tests/unit/select_test.coffee

```diff
@@ -197,3 +199,42 @@ test 'Test query matching', ->
   select.set 'query', 'bana$ n'
   equal(select.get('filteredContent').length, 1,
     'duplicated spaces in the source string should be removed before matching')
+
+test "Show empty content view if content is empty", ->
+  expect 2
+
+  select = @subject
+    content: []
+    optionLabelPath: 'name'
+    optionValuePath: 'code'
+    classNames: 'select-class-name'
+
+  @append()
+
+  selectElement = select.$()
+  openDropdown(selectElement)
+  andThen ->
+    ok isPresent(emptyContentSelector, selectElement),
+      'Empty content view displayed'
```

Comment:
this assertion is a bit misleading as it doesn't clearly test `emptyContentView`